docker.env

```diff
@@ -4,6 +4,8 @@ POSTGRES_HOST=postgres
 JDBC_LOG_LEVEL=DEBUG
 WEB_LOG_LEVEL=DEBUG
 APP_LOG_LEVEL=DEBUG
+MAIL_USERNAME=abialiauski.alerter@gmail.com
+MAIL_PASSWORD=jypsojwddawmkcfq
```

Comment:
@hizmailovich can you encrypt it or store more safely?

Reply:
JavaMailSender doesn't support encrypted password to set, but we can use base64 decoder for username and password, pushed